### ISSUE TITLE
[Packaging Request] ryujinx-ldn

### ISSUE THREAD

**Issue by IceDBorn** [NixOS/nixpkgs#187020]
**Project description**
Ryujinx's internet multiplayer build, even supporting interconnectivity with Switch Consoles.

**Metadata**

* homepage URL: https://www.patreon.com/posts/introducing-ldn2-45268370
* source URL: https://github.com/Ryujinx/Ryujinx
* license: MIT
* platforms: linux

**Comment by IvarWithoutBones** [comment]
Hi! I am one of the maintainers of the existing Ryujinx package and thought I'd give my two cents.

The source code for Ryujinx LDN is currently not released, because of that we would have to `patchelf` a precompiled binary which can be quite a pain with dotnet application if I recall correctly.

I've also talked to upstream devs about this version before, if i recall correctly they said it needs a rework and won't be releasing the source code before that's done.

I'm personally inclined to wait with packaging this build until that's done.

**Comment by IvarWithoutBones** [comment]
cc @06kellyjac

**Comment by 06kellyjac** [comment]
I agree with @IvarWithoutBones 

The release tar.gz would need patching not only of the shared objects the binary is linked to but also the other libraries that are loaded at runtime like libSDL2.so & libsoundio.so, but if someone does get it working then cool

Also I wouldn't really call it MIT licensed until the code is actually public, I'd license it as `unfree` or `unfreeRedistributable` if we get permission

**Comment by IceDBorn** [comment]
I agree with you guys, I was not aware that the source code was not public yet. Do I close or keep this issue open?

**Comment by 06kellyjac** [comment]
you can keep it open so it's visible if you don't mind having an issue open

**Comment by IceDBorn** [comment]
I'll just keep this issue open, others can see the conversation we had and understand the reason this is not a native package yet.
